Balancetweak

buckly90 opened the pull request:
Made some small tweaks to hopefully make the game more enjoyable.

- Limited all patch lighting to a maximum of 100% No more 150% or above.
- Buffed photosynthesis a little to make up for the lower limits of light.
- Rearranged membrane stats in an attempt to make each type more unique and viable.
- Buffed respiration parts. You can now support a flagellum with as little as two metabolosomes. (Your still gonna need alot if you want to be complex.)

These changes will be revisited once the spawn system is finalized and I am able to freely tweak compound rates. For now, the changes are aimed to hopefully make the game more enjoyable in it's current state.

buckly90 commented:
I'm not currently sure where the code for the membrane tooltips resides, and I need to change that so the editor displays the appropriate stats for membranes.

hhyyrylainen commented on simulation_parameters/microbe_stage/biomes.json:
```diff
@@ -25,12 +25,12 @@
                     "name": "Floating Hazard",
                     "meshes": [
                         {
-                            "scenePath": "res://src/microbe_stage/ToxinCloud.tscn"
```
This file looks like all the changes are backwards, this reverts the change to moving to using particles for the toxins.

hhyyrylainen reviewed (changes requested):
I think the tooltips might be in ToolTipManager.tscn, though @Kasterisk probably knows better

Kasterisk commented:
Yeah all the custom tooltips are currently in ToolTipManager.tscn. The modifier values are hardcoded on the tooltip gui so you need to tweak the the gui labels text to change them.

buckly90 commented:
Tooltips have been changed, I went ahead and converted the remaining decimal values to percentage as well, I also fixed the issue with my commit reverting the changes to toxins.

hhyyrylainen reviewed (approved):
Looks like there are no longer any problems with unintended changes being included.
@Kasterisk could you check the GUI changes?
Other than that we might want to hear balance feedback from other devs.

Kasterisk reviewed (approved):
Looks good

hhyyrylainen commented:
I'll merge this to prepare for tomorrow's release candidate.
